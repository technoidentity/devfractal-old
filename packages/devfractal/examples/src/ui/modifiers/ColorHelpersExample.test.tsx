import React from 'react'
import renderer, { ReactTestRenderer } from 'react-test-renderer'
import { Color } from './ColorHelpersExample'

test('Color Helpers Example', () => {
  const component: ReactTestRenderer = renderer.create(<Color />)
  expect(component.toJSON()).toMatchSnapshot()
})
