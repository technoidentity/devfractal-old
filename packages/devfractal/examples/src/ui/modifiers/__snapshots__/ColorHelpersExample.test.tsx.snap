// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Color Helpers Example 1`] = `
Array [
  <section
    className="section"
  >
    <h1
      className="title"
    >
      Text Color
    </h1>
    <div
      className="box"
    >
      <div
        className="has-text-black"
      >
        black
      </div>
      <div
        className="has-text-primary"
      >
        primary
      </div>
      <div
        className="has-text-success"
      >
        success
      </div>
      <div
        className="has-text-warning"
      >
        warning
      </div>
      <div
        className="has-text-link"
      >
        link
      </div>
      <div
        className="has-text-dark"
      >
        dark
      </div>
      <div
        className="has-text-info"
      >
        info
      </div>
      <div
        className="has-text-white"
      >
        white
      </div>
      <div
        className="has-text-danger"
      >
        Danger
      </div>
      <div
        className="has-text-black-bis"
      >
        black-bis
      </div>
      <div
        className="has-text-black-ter"
      >
        black-ter
      </div>
      <div
        className="has-text-grey-darker"
      >
        grey-darker
      </div>
      <div
        className="has-text-grey-dark"
      >
        grey-dark
      </div>
      <div
        className="has-text-grey"
      >
        grey
      </div>
      <div
        className="has-text-grey-light"
      >
        grey-light
      </div>
      <div
        className="has-text-grey-lighter"
      >
        grey-lighter
      </div>
      <div
        className="has-text-white-ter"
      >
        white-ter
      </div>
      <div
        className="has-text-white-bis"
      >
        White-bis
      </div>
    </div>
  </section>,
  <section
    className="section"
  >
    <h1
      className="title"
    >
      Background Color
    </h1>
    <div
      className="box"
    >
      <div
        className="has-background-black"
      >
        black
      </div>
      <div
        className="has-background-primary"
      >
        primary
      </div>
      <div
        className="has-background-success"
      >
        success
      </div>
      <div
        className="has-background-warning"
      >
        warning
      </div>
      <div
        className="has-background-link"
      >
        link
      </div>
      <div
        className="has-background-dark"
      >
        dark
      </div>
      <div
        className="has-background-info"
      >
        info
      </div>
      <div
        className="has-background-white"
      >
        white
      </div>
      <div
        className="has-background-danger"
      >
        Danger
      </div>
      <div
        className="has-background-black-bis"
      >
        black-bis
      </div>
      <div
        className="has-background-black-ter"
      >
        black-ter
      </div>
      <div
        className="has-background-grey-darker"
      >
        grey-darker
      </div>
      <div
        className="has-background-grey-dark"
      >
        grey-dark
      </div>
      <div
        className="has-background-grey"
      >
        grey
      </div>
      <div
        className="has-background-grey-light"
      >
        grey-light
      </div>
      <div
        className="has-background-grey-lighter"
      >
        grey-lighter
      </div>
      <div
        className="has-background-white-ter"
      >
        white-ter
      </div>
      <div
        className="has-background-white-bis"
      >
        White-bis
      </div>
    </div>
  </section>,
]
`;
